openshift.spec

```diff
@@ -122,6 +122,9 @@ install -m 0644 -t %{buildroot}/usr/lib/tuned/openshift-node tuned/openshift-nod
 
 %post master
 %systemd_post %{basename:openshift-master.service}
+HOSTNAME=`hostname -f`
+echo "#PUBLICMASTER specifies the hostname where external hosts would reach the API and Console" >> /etc/sysconfig/openshift-master
+echo "PUBLICMASTER=https://${HOSTNAME}:8443" >> /etc/sysconfig/openshift-master
```

Comment:
@sdodson despite what I said earlier, not crazy about doing this. i think @dobbymoodge raised a valid point that if we can't determine a good default, we probably shouldn't put anything at all. taking whatever the hostname is at RPM install time doesn't seem like a good default, and it's hard to think what one could be. The defaults of localhost and NIC IP should already be covered. I think maybe we should stick with not putting --public-master explicitly in the unit/sysconfig file.